behaviour/graql/language/define.feature

```diff
@@ -338,66 +744,352 @@ Feature: Graql Define Query
       | PART_TIME | EMPLOYER |
 
 
-  Scenario: define a type as abstract errors if has non-abstract parent types (?)
+  ##############
+  # ATTRIBUTES #
+  ##############
 
-  Scenario: define a type as abstract creates an abstract type
 
-  Scenario: define a regex on an attribute type, attribute type queryable by regex value
+  Scenario: define attribute creates a type
+    Given graql define
+      """
+      define favourite-food sub attribute, value string;
+      """
+    Given the integrity is validated
+    When get answers of graql query
+      """
+      match $x type favourite-food; get;
+      """
+    Then concept identifiers are
+      |     | check | value          |
+      | FAV | label | favourite-food |
+    Then uniquely identify answer concepts
+      | x   |
+      | FAV |
 
-  Scenario: define a rule creates a rule (?)
 
-  Scenario: define a sub-role using 'as' is visible from children (?)
+  Scenario: define attribute subtype creates child of its parent type
+    Given graql define
+      """
+      define first-name sub name;
+      """
+    Given the integrity is validated
+
+    When get answers of graql query
+      """
+      match $x sub name; get;
+      """
+    Then concept identifiers are
+      |        | check | value      |
+      | NAME   | label | name       |
+      | F_NAME | label | first-name |
+    Then uniquely identify answer concepts
+      | x      |
+      | NAME   |
+      | F_NAME |
+
+
+  Scenario: define attribute subtype inherits 'value' from its parent
+    Given graql define
+      """
+      define first-name sub name;
+      """
+    Given the integrity is validated
+
+    When get answers of graql query
+      """
+      match $x type first-name, value string; get;
+      """
+    Then concept identifiers are
+      |        | check | value      |
+      | F_NAME | label | first-name |
+    Then uniquely identify answer concepts
+      | x      |
+      | F_NAME |
+
+
+  Scenario: define attribute type throws if you don't specify 'value'
+    Then graql define throws
+      """
+      define colour sub attribute;
+      """
+
+
+  Scenario: define attribute type throws if 'value' is invalid
+    Then graql define throws
+      """
+      define colour sub attribute, value rgba;
+      """
+
+
+  @ignore
```

Comment:
make this into an issue in `verification` so we don't for get :)

Reply:
https://github.com/graknlabs/verification/issues/46